docs/application/dotnet/get-started/nui/quickstart.md

```diff
@@ -269,7 +273,7 @@ You can add your code in **SampleNUIApp.cs** file, as follows:
 
     1.  Copy the application .tpk file from the project binary path to the SDB tool path:
 
-        - The `SampleNUIApp` tpk file(`org.tizen.example.SampleNUIApp-1.0.0.tpk`) is located in `<ProjectPath>\SampleNUIApp\SampleNUIApp\bin\Debug\tizen50`.
+        - The `SampleNUIApp` tpk file(`org.tizen.example.SampleNUIApp-1.0.0.tpk`) is located in `<ProjectPath>\SampleNUIApp\SampleNUIApp\bin\Debug\tizen{version}`.
```

Comment:
`SampleNUIApp` same comment on using bold
The **SampleNUIApp** TPK file(`org.tizen.example.SampleNUIApp-1.0.0.tpk`) is located in `<ProjectPath>\SampleNUIApp\SampleNUIApp\bin\Debug\tizen{version}`.